main.go

```diff
@@ -72,15 +87,27 @@ func main() {
 	} else {
 		isForm := opts.Has("F")
 		if pretty || terminal.IsTerminal(stdoutFd) {
-			inputWriter = &formatter.JSON{
-				Out:    inputWriter,
-				Scheme: formatter.DefaultColorScheme,
-			}
-			// Format/colorize JSON output if stdout is to the terminal.
-			stdout = &formatter.JSON{
-				Out:    stdout,
-				Scheme: formatter.DefaultColorScheme,
+			// if disbleColor is true, then don't set Scheme.
```

Comment:
Would be better to declare a `scheme` variable instead of `disableColor` + duplication code.

Reply:
Sorry my bad. Updated it.